src/thruster.cpp

```diff
@@ -144,33 +150,36 @@ void Thruster::UpdateBuoyancy()
 
 void Thruster::UpdateVisualizers()
 {
-    for(int i=0; i < num_thrusters; i++)
+    if(visualize_thrusters)
```

Comment:
Maybe a better place for this is where updateVisualizers() is called on line 241.